core/templates/layout.public.php

```diff
@@ -80,7 +80,7 @@
 		</div>
 	</header>
 
-	<main id="content" class="app-<?php p($_['appid']) ?>">
+	<div id="content" class="app-<?php p($_['appid']) ?>">
```

Comment:
Should we maybe just add a main for native apps?

```suggestion
	<div id="content" class="app-<?php p($_['appid']) ?>">
		<main>
```